incubator/kafka/templates/statefulset.yaml

```diff
@@ -224,6 +224,10 @@ spec:
           name: {{ include "kafka.fullname" . }}-metrics
           {{- end }}
       {{- end }}
+      {{- if .Values.securityContext }}
+      securityContext:
+        {{- .Values.securityContext | toYaml | nindent 8 }}
```

Comment:
have it left-justified and then indent - makes it easier and consistent with the rest of the manifest(s).

Reply:
Updated as requested